Client/Frontend/Widgets/AutocompleteTextField.swift

```diff
@@ -293,3 +293,11 @@ class AutocompleteTextField: UITextField, UITextFieldDelegate {
     }
 
 }
+
+extension AutocompleteTextField : MenuHelperInterface {
```

Comment:
The CI menitoned that there shouldnt be a space between `AutocompleteTextField` and the `:`